docs/index.rst

```diff
@@ -0,0 +1,33 @@
+======================
+Qiskit-QuantumInspire
+======================
+
+Welcome to the documentation of **Qiskit-QuantumInspire**, a Qiskit plugin for the Quantum Inspire platform.
+
+.. toctree::
+   :hidden:
```

Comment:
Do we keep this hidden because we only want to show it in the sidebar? Might not hurt to also list the contents on the home page to make it less empty.

Reply:
Yes, the homepage is a bit empty, but if this isn't hidden it becomes ugly instead 😅